doc/sphinx/source/tutorial.rst

```diff
@@ -331,16 +343,27 @@ Then you still need to format the string with your variable:
 
 .. code-block:: lua
 
-   print pretty_plurals_string:format(number_of_worlds)
+   print pretty_plurals_string:bformat(number_of_worlds)
 
-If you have a string with multiple numbers in it that would trigger plural forms, split it into separate strings that you can fetch with ``ngettext``. You can then combine them with ``bformat`` and ordered placeholders.
+If you have a string with multiple numbers in it that would trigger plural
+forms, split it into separate strings that you can fetch with ``ngettext``.
+You can then combine them with ``bformat`` and ordered placeholders.
 
 
 Handling Long Strings
 ^^^^^^^^^^^^^^^^^^^^^
 
-If you have a really long string, e.g. a dialog stretching over multiple sentences, check if there is a logical place where you could split this into two separate strings for translators. We don't have a "break after x characters" rule for this; please use common sense here. It is easier for translators to translate smaller chunks, and if you should have to change the string later on, e.g. to fix a typo, you will break less translations. The strings will be put into the translation files in the same order as they appear in the source code, so the context will remain intact for the translators.
-
-Also, please hide all formatting control characers from our translators. This includes HTML tags as well as new lines in the code! For an example, have a look at ``data/campaigns/atl01.wmf/scripting/texts.lua``
+If you have a really long string, e.g. a dialog stretching over multiple
+sentences, check if there is a logical place where you could split this into
+two separate strings for translators. We don't have a "break after x characters"
+rule for this; please use common sense here. It is easier for translators to
+translate smaller chunks, and if you should have to change the string later on,
+e.g. to fix a typo, you will break less translations. The strings will be put
+into the translation files in the same order as they appear in the source code,
+so the context will remain intact for the translators.
+
+Also, please hide all formatting control characters from our translators. This
+includes HTML tags as well as new lines in the code! For an example, have a
+look at ``data/campaigns/atl01.wmf/scripting/texts.lua``.
```

Comment:
Also add a note that one must not simply concatenate two sentences but instead use bformat or a localized-concatenation function (don't remember what it's called and where it's defined), unless there's a paragraphdivider or other richtext markup between the sentences.

Reply:
I have not enough knowledge about this localization stuff to write such a note.

Changed HTML with a link to richtext.lua.

Reply:
```
Note that simply concatenating two translatable strings with ``..``
does not work in all languages. To achieve the desired effect, use
the function :meth:`join_sentences` with the two sentences as
arguments, or :meth:`paragraphdivider` if you wish the text to
continue on a new paragraph.

Never assemble a single localized sentence with the ``..``
operator. Use :meth:`string.bformat` with appropriate
placeholder substitutions for such cases.
```

Reply:
Thanks, done.